# Problem 1
"Find the sum of all the multiples of 3 or 5 below 1000."

In [5]:
sum = 0
for number in range(1000):
    if number % 3 == 0 or number % 5 == 0:
        sum = sum + number

print(sum)
        

233168


My solution to problem 1 is 233168.

This solution is correct!